src/redux/sagas/root-saga.tsx

```diff
@@ -0,0 +1,6 @@
+import { fork } from 'redux-saga/effects';
+import firestoreSaga from './firestore-saga';
+
+export default function* rootSaga() {
+  yield fork(firestoreSaga);
```

Comment:
I would say one-line this but i'm unsure about `yeild`ing on a single line...

Reply:
I'd keep it as is since this the root saga where I'd call other sagas as well. It's just that `firestoreSaga` is the only one that exists at the moment.

Reply:
Thanks for that 👍